xline-client/src/types/auth.rs

```diff
@@ -36,7 +36,7 @@ impl From<AuthenticateRequest> for xlineapi::AuthenticateRequest {
 }
 
 /// Request for `Authenticate`
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
What is this `PartialEq` for?

Reply:
explained in https://github.com/xline-kv/Xline/pull/424#discussion_r1326883906